modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc

```diff
@@ -5920,6 +5920,132 @@ Total database accesses: 5, total allocated memory: 1488
 ======
 
 
+[[query-plan-stateful-shortest-path-into]]
+== StatefulShortestPath(Into)
+// StatefulShortestPath(Into)
+
+The `StatefulShortestPath(Into)` operator finds shortest paths between two previously matched node variables.
+It uses a bidirectional breath first search (BFS) algorithm, which performs two BFS invocations at the same time, one from the left boundary node and one from the right boundary node.
+Once a node is found by both BFS invocations, which indicates that it can be reached from both boundary nodes, the algorithm successfully terminates.
+No valid path exists between both boundary nodes when both BFS invocations terminate without any overlap in the nodes that both invocations found.
+
+.StatefulShortestPath(Into)
+======
+
+.Query
+[source, cypher]
+----
+PROFILE
+MATCH
+  (chris:Person {name: 'Chris'}),
+  (stefan:Person {name: 'Stefan'})
+CALL {
+  WITH chris, stefan
+  MATCH
+    p = ALL SHORTEST (chris)(()-[]-()-[]-()){1,}(stefan)
+  RETURN p
+}
+RETURN p
```

Comment:
I would suggest changing the `CREATE RANGE INDEX range_person_name FOR (p:Person) ON (p.name)` to a uniqueness constraint instead.

Like that, you hopefully will not need the `CALL`-workaround in this example.